app/javascript/authentication/SignInForm.tsx

```diff
@@ -94,7 +94,11 @@ const SignInFormCard = ({
           {requestError}
         </AlertBox>
       )}
-      <SiteAlert type="success" />
+      {sessionStorage.getItem("alert_message_success") ? (
+        <SiteAlert type="success" />
+      ) : (
+        <SiteAlert type="secondary" />
```

Comment:
Will there ever be a case where there is nothing in sessionStorage under `alert_message_secondary`